x/marker/keeper/keeper_test.go

```diff
@@ -1488,3 +1492,91 @@ func TestUpdateSendDenyList(t *testing.T) {
 		})
 	}
 }
+
+func TestReqAttrBypassAddrs(t *testing.T) {
+	// Tests both GetReqAttrBypassAddrs and IsReqAttrBypassAddr.
+	expectedNames := []string{
+		authtypes.FeeCollectorName,
+		rewardtypes.ModuleName,
+		quarantine.ModuleName,
+		govtypes.ModuleName,
+		distrtypes.ModuleName,
+		stakingtypes.BondedPoolName,
+		stakingtypes.NotBondedPoolName,
+	}
+
+	incByte := func(b byte) byte {
+		if b == 0xFF {
+			return 0x00
+		}
+		return b + 1
+	}
+
+	app := simapp.Setup(t)
+
+	for _, name := range expectedNames {
+		t.Run(fmt.Sprintf("get: contains %s", name), func(t *testing.T) {
+			expAddr := authtypes.NewModuleAddress(name)
+			actual := app.MarkerKeeper.GetReqAttrBypassAddrs()
+			assert.Contains(t, actual, expAddr, "GetReqAttrBypassAddrs()")
+		})
+	}
+	t.Run("get: only has expected entries", func(t *testing.T) {
+		// This assumes each expectedNames test passed. This is designed to fail if a new entry
+		// is added to the list (in app.go). When that happens, update the expectedNames with
+		// the new entry so it's harder for it to accidentally go missing.
+		actual := app.MarkerKeeper.GetReqAttrBypassAddrs()
+		assert.Len(t, actual, len(expectedNames), "GetReqAttrBypassAddrs()")
+	})
+
+	t.Run("get: called twice equal but not same", func(t *testing.T) {
+		expected := app.MarkerKeeper.GetReqAttrBypassAddrs()
+		actual := app.MarkerKeeper.GetReqAttrBypassAddrs()
+		if assert.Equal(t, expected, actual, "GetReqAttrBypassAddrs()") {
+			if assert.NotSame(t, expected, actual, "GetReqAttrBypassAddrs()") {
+				for i := range expected {
+					assert.NotSame(t, expected[i], actual[i], "GetReqAttrBypassAddrs()[%d]", i)
+				}
+			}
+		}
+	})
+
+	t.Run("get: changes to result not reflected in next result", func(t *testing.T) {
```

Comment:
It feels like these addresses should probably be "sealed" when the marker module is created.

Reply:
They almost are since there's no keeper functionality for changing them. The only way they could get changed after keeper creation would be to alter the list provided to the constructor, but that's pretty limiting.

Even still, I updated `NewKeeper` to store a deep copy of the provided slice instead of the provided slice directly. Now, the only way to change it after keeper creation would be to add functionality to the keeper.